validator/src/main.rs

```diff
@@ -1577,6 +1577,14 @@ pub fn main() {
     )
     .unwrap_or_default();
 
+    validator_config.generator_config =
```

Comment:
can this just be `None` in the public repo?

then we can take the arg out of public also, we really don't want this stuff exposed to operators (even this pr is a bit risky in its current form)

Reply:
Yep, we can do that. Downside of removing the arg is more divergence/conflicts between repos - obviously in the arg parsing area (but we discussed how to reduce that down to a couple function calls a la nonce) but also in the net scripts that pass through accounts path

Reply:
Removed the arg